lib/reek/context/code_context.rb

```diff
@@ -9,7 +9,7 @@ module Context
     # code element. CodeContexts form a tree in the same way the code does,
     # with each context holding a reference to a unique outer context.
     #
-    # :reek:TooManyMethods: { max_methods: 18 }
+    # :reek:TooManyMethods: { max_methods: 19 }
```

Comment:
Kind of sucks, but refactoring CodeContext is out of the scope for this PR and I'd rather address this separately.